gprofiler/profilers/java.py

```diff
@@ -302,6 +304,18 @@ def read_output(self) -> Optional[str]:
             help="Select async-profiler's mode: 'cpu' (based on perf_events & fdtransfer) or 'itimer' (no perf_events)."
             " Defaults to '%(default)s'.",
         ),
+        ProfilerArgument(
+            "--java-async-profiler-safemode",
+            dest="java_async_profiler_safemode",
+            type=int,
+            default=0,
+            choices=range(0, 128),
+            metavar="[0-127]",
+            help="Controls the 'safemode' parameter passed to async-profiler. This is parameter denotes multiple"
+            " bits that describe different stack recovery techniques which async-profiler uses (see StackRecovery"
+            " enum in async-profiler's code)."
```

Comment:
Give a link or at least the name of the file?

Reply:
Added